tratto/src/main/java/star/tratto/identifiers/JavadocFormat.java

```diff
@@ -0,0 +1,18 @@
+package star.tratto.identifiers;
+
+public enum JavadocFormat {
+    CLASS_PREFIX("/**"),
+    CLASS_SUFFIX("*/"),
+    METHOD_PREFIX("\t/**"),
```

Comment:
Why is there a tab there?  Is `JavadocFormat` for input or output?  If for output, I would prefer not to put tabs, but rather spaces, in the output.